src/components/Select/Select.jsx

```diff
@@ -22,12 +22,17 @@ export default class Select extends React.Component {
   static sameItems = (itemsA, itemsB) =>
     itemsA.length === itemsB.length &&
     itemsA.every(
-      (el, ix) => el.id === itemsB[ix].id && el.title === itemsB[ix].title
+      (el, ix) =>
+        el &&
+        itemsB[ix] &&
+        el.id === itemsB[ix].id &&
+        el.title === itemsB[ix].title
     );
 
   state = {
     isOpen: this.props.isOpen,
     items: this.props.items || [],
+    selectedItems: this.props.items || [],
```

Comment:
we're adding another items array here to the state in order to use it later when filtering/searching the items